assets/src/js/header.js

```diff
@@ -227,5 +227,19 @@ export const setupHeader = () => {
     }
   });
 
+  /**
+   * Setup max height to submenues.
+   * The limits should be 6 for scrolleable and 10 for maximum into the list
```

Comment:
We settled for soft limitations in [PLANET-6508](https://github.com/greenpeace/planet4-master-theme/pull/1577), you can implement those in the menu editor like this (it includes a fix for a bug in the current code :grimacing: )
```diff
diff --git a/assets/src/js/menu_editor.js b/assets/src/js/menu_editor.js
index a6e98cef..4dd39abb 100644
--- a/assets/src/js/menu_editor.js
+++ b/assets/src/js/menu_editor.js
@@ -160,7 +160,7 @@ const menuEditorRestrictions = () => {
     return locations.map(location => {
       return document.querySelector(`input[name="menu-locations[${location}]"]`).checked
         ? location : null;
-    })[0] || null;
+    }).filter(l => l)[0] || null;
   };
 
   /**
diff --git a/src/AdminAssets.php b/src/AdminAssets.php
index f4a6a094..7fb7dcd8 100644
--- a/src/AdminAssets.php
+++ b/src/AdminAssets.php
@@ -24,8 +24,9 @@ class AdminAssets {
 		$theme_dir       = get_template_directory_uri();
 		$menus           = get_nav_menu_locations();
 		$navbar_location = 'navigation-bar-menu';
+		$donate_location = 'donate-menu';
 
-		if ( ! isset( $menus[ $navbar_location ] ) ) {
+		if ( ! isset( $menus[ $navbar_location ] ) && ! isset( $menus[ $donate_location ] ) ) {
 			return;
 		}
 
@@ -44,6 +45,11 @@ class AdminAssets {
 					],
 				],
 			],
+			$donate_location => [
+				'maxDepth' => 0,
+				'maxItems' => 5,
+				'maxChars' => 18,
+			],
 		];
 
 		$conf = apply_filters( 'planet4_menu_config', $default_conf );
```

Reply:
Thanks for point me this! 
I've added the sub menu setup just for setting the `scrolleable` class in cases where the list would have more than 6 items. See what the ticket mentions:
> A drop down menu should show 6 visible items and have a scroll bar when there are more than 6 items on desktop screens. 

Let me know if I'm in the correct way!

Reply:
Sounds good :+1: and in my diff `maxItems` should be `6` then I guess

Reply:
Yep! I've already changed that